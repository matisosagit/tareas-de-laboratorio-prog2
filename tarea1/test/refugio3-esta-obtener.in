crearRefugio

# creo cuatro perros para ingresarlos al refugio 

crearFecha 24/7/2020
crearPerro 1 Nala 10 4 Beige beteado, grande
agregarEnRefugio

crearFecha 7/7/2020
crearPerro 2 Tito 12 3 Blanco, chico
agregarEnRefugio

crearFecha 8/10/2021
crearPerro 3 Tarzan 1 10 Marron oscuro, mediano
agregarEnRefugio

crearFecha 28/5/2018
crearPerro 4 Paco 13 5 Gris claro, grande
agregarEnRefugio

# chequ√©o si existen diferentes ids de perros y los obtengo

estaEnRefugio 5
estaEnRefugio 2
estaEnRefugio 3
estaEnRefugio 0

obtenerDeRefugio 2

liberarRefugio
Fin
