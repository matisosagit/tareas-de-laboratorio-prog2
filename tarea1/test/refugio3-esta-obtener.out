1>2>#  creo cuatro perros para ingresarlos al refugio .
3>4>5>6>7>8>9>10>11>12>13>14>15>#  chequÃ©o si existen diferentes ids de perros y los obtengo.
16>El perro con id 5 NO esta en el refugio.
17>El perro con id 2 esta en el refugio.
18>El perro con id 3 esta en el refugio.
19>El perro con id 0 NO esta en el refugio.
20>Perro 2
Nombre: Tito
Edad: 12
Descripcion:  Blanco, chico
Fecha de ingreso: 7/7/2020
Vitalidad: 3
21>22>Fin.
